CHANGELOG.md

```diff
@@ -46,6 +46,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Fix issue with feature flags where default value may not be honored ([#12849](https://github.com/opensearch-project/OpenSearch/pull/12849))
 - Fix UOE While building Exists query for nested search_as_you_type field ([#12048](https://github.com/opensearch-project/OpenSearch/pull/12048))
 - Client with Java 8 runtime and Apache HttpClient 5 Transport fails with java.lang.NoSuchMethodError: java.nio.ByteBuffer.flip()Ljava/nio/ByteBuffer ([#13100](https://github.com/opensearch-project/opensearch-java/pull/13100))
+- Fix implement mark() and markSupported() in class FilterStreamInput ([#13098](https://github.com/opensearch-project/OpenSearch/pull/13098))
```

Comment:
remove this.

Reply:
I can't see this as a new change in files changed.  Is it still showing as new line added to you? 
<img width="1349" alt="Screenshot 2024-04-11 at 11 36 57 AM" src="https://github.com/opensearch-project/OpenSearch/assets/78371932/2cead117-d648-4ef2-91c6-a326c6375ebc">